.github/workflows/build.yml

```diff
@@ -70,32 +70,32 @@ jobs:
           github-token: ${{ secrets.BOT_TOKEN }}
           script: |
             const helper = require('./.github/util/workflow-helper.js')({github, context});
-            
+
```

Comment:
I can undo these formatting/whitespace changes.

Reply:
Looks like Intellij did this automatically when resolving merge conflicts, so I'm going to leave it so we don't have to deal with it later.